ps_mbo.php

```diff
@@ -163,7 +163,17 @@ public function __construct()
     public function install()
     {
         return parent::install()
-            && $this->registerHook(static::HOOKS)
+            && $this->registerHook(static::HOOKS);
+    }
+
+    /**
+     * Enable Module.
+     *
+     * @return bool
+     */
+    public function enable($force_all = false)
+    {
+        return parent::enable($force_all)
```

Comment:
It looks like a bad design from the Core. The Core itself should be able to check if the module is installed or not, then display links :thinking:

Reply:
@PierreRambaud You meant enable?

Reply:
Yes :sweat_smile: I mean installed and enabled :)

Reply:
@PierreRambaud actually, the problem comes from the fact that ps_mbo disables core links during its installation and re-enables them only when uninstalling. I'm not sure that's the core role to use a disabled link when the module replacing those links is disabled. WDYT?

Reply:
I agree, your change is the best one right now. I was just thinking about this behavior should be handle by the Core and not the module :sweat_smile: